packages/ui/utils/getFirstLetter.ts

```diff
@@ -0,0 +1,12 @@
+export const getFirstLetter = (name: string): string => {
+  // When user image is set to NULL we use first letter as placeholder for image (Like in Google)
+  let index = 0;
+  while (index < name.length) {
+    const letter = name.charAt(index);
+    if (letter.match(/[a-zA-Z0-9]/)) {
+      return letter.toUpperCase();
+    }
```

Comment:
I think the while loop here is completely an overkill its not needed in my opinion.
you could easily get the 1st char like this
 
Array.from(name)[0] or name.substring(0,1)

Reply:
Oh okay, thanks for suggestion)

Reply:
But about loop, the point here is that it checks if first letter is not a special character and if name is completely one special character it will return 'G'